src/js/worker/core/player.js

```diff
@@ -884,18 +922,42 @@ function generate(
     scoutingRank: number,
 ): PlayerWithoutPid {
     let ratings;
+	
+	var PD = require("probability-distributions");	
+	let realHeight;
+	let predetHgt;
+    
+	// RealHeight will never go lower than 4'0" or higher than 9'6".  (Very unlikely to ever see those anyway.)	
+	realHeight = helpers.bound(PD.rgamma(1,472.0530111,5.981338), 48, 114);
+	
+	// Ugly hack: extreme outliers (7'6"+) are about right, but it's generating too many guys 7-3 to 7-5
+	// So lose two inches on some of them
```

Comment:
I don't think it would work to find new parameters.  I used nearly 2000 data points to get the parameters - I don't think it's going to get much better.  The problem is that the real-life height distribution makes a funny turn at 7'1".  It drops off severely.  Based on the curve of everything up through 7'0", there *should* be a lot more players at 7'1", 7'2", etc.  There aren't, but the best still overestimates them.

An alternative would be to truncate the Gamma function at 7'1" and then control the outliers manually.  If someone over 7'1" is generated, X% of the time drop them (to account for the fact that too many 7'2"-7'5" players are being generated, and otherwise use an entirely separate distribution to decide the height of the outlier.  Although if we're truncating it, might as well use a normal distribution - the whole point of the gamma distribution was that it was the one that got the rate right for 7'6"+.  I hadn't noticed until testing that the 7'2"-7'5" players were coming out too often.   Normal seemed okay except for making the big outliers too rare.